tests/bash/comp-tests.bash

```diff
@@ -192,25 +241,28 @@ fi
 # Measure speed of execution without descriptions (for both v1 and v2)
 _completionTests_timing "testprog manycomps " 0.2 "no descriptions"
 
-# Test other bash completion types with descriptions disabled.
-# There should be no change in behaviour when there are no descriptions.
-# The types are: menu-complete/menu-complete-backward (COMP_TYPE == 37)
-# and insert-completions (COMP_TYPE == 42)
-COMP_TYPE=37
-_completionTests_verifyCompletion "testprog prefix nospace b" "bear bearpaw" nospace
-_completionTests_verifyCompletion "testprog prefix nofile b" "bear bearpaw" nofile
+# COMP_TYPE does not get set by bash 3
+if [ $BASH_VERSINFO != 3 ]; then
```

Comment:
It turns out that there is no point in running the tests that set `COMP_TYPE` for bash 3 because `COMP_TYPE` only gets set by bash 4.  So the tests were passing because we were setting `COMP_TYPE` ourselves, but it is not representative of what bash 3 does.  So I have disable then for bash 3.